src/ContextualPopup/ContextualPopup.js

```diff
@@ -222,6 +229,18 @@ module.exports = kind(
 	initComponents: function () {
 		this.createChrome(this.tools);
 		Popup.prototype.initComponents.apply(this, arguments);
+		this.syncProperties();
+	},
+
+	/**
+	* Sync spotlightModal and modal properties on creation time.
+	*
+	* @private
+	*/
+	syncProperties: function () {
+		if (this.modal == this.spotlightModal) return;
+		if (this.hasOwnProperty('spotlightModal')) this.modal = this.spotlightModal;
```

Comment:
I'm copying comment from Jira. 

---
Hi Aaron Tam,

Current default properties are:
- enyo.Popup.modal = false
- moon.ContextualPopup.spotlightModal = false

If user can specify both of modal and spotlightModal on components block. I can think of below four cases.
1) User can override modal -> sync from modal to spotlightModal
2) User can override spotlightModal -> sync from spotlightModal to modal
3) User can override both modal and spotlightModal with same value -> No sync needed
4) User can override both modal and spotlightModal with different value -> ???

If modal and spotlightModal is doing the same, I think we don't even need to open spotlightModal as published property. But, just change it internally. How do you think?

There is another way of syncing. Lookup hasOwnProperty and decide the direction of sync.
But, in case of 4), we need to decide the default sync direction like 'sync from spotlightModal to modal' with some warning to developer.
---

The code is comes from the last comment.